dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -598,18 +598,19 @@ public List<BulkEditChange> runImport(Context c, boolean change,
                 changes.add(whatHasChanged);
             }
 
-            if (change) {
-                //only clear cache if changes have been made.
-                c.uncacheEntity(wsItem);
-                c.uncacheEntity(wfItem);
-                c.uncacheEntity(item);
+            if (change && (rowCount % configurationService.getIntProperty("bulkedit.change.commit.count") == 0)) {
```

Comment:
We may want to add the default value of `bulkedit.change.commit.count` into this line by changing it to be:
```
configurationService.getIntProperty("bulkedit.change.commit.count", 100)
```

Otherwise, if someone is missing this setting in their config or accidentally comments it out, the default will be *zero*, which likely will cause odd behaviors (or errors) on this line if you are dividing by zero.